### ISSUE TITLE
Both of remote backup options doesn't work (Google Drive and Dropbox)

### ISSUE THREAD

**Issue by xscd** [LN-Zap/zap-desktop#3309]
Currently both of the remote backup options doesn't work. 

## Description

Google Drive tells that the browser which Zap opens is insecure and unsupported. Dropbox has suspended activity of Zap and needs a manual review. 

## Expected Behavior

Both remote backup options should work. 

## Actual Behavior

Both remote backup options doesn't work. 

Drive:
![Image](https://files.catbox.moe/p0idk7.png)

Dropbox:
![Image](https://files.catbox.moe/ysua9i.png)

## Possible Fix

Drive recommends this: https://support.google.com/accounts/answer/7675428

Dropbox, no idea probably contacting them as Zap? 

## Steps to Reproduce

- 1. Create new waller
- 2. Select Drive or Dropbox as remote backup option
- 3. Try to enter login credentials 
- 4. Good Luck! 

## Context

I want my channel state backups in remote. I don't trust my HDD since it makes weird noises and will probably fail soon. I can't currently use the wallet. 

## Your Environment

I use Linux and the AppImage edition (don't know if there are any *.deb which wouldn't work anyway because I don't use Debian or Ubuntu anyway) 

- Zap version: Zap-linux-x86_64-v0.6.2-beta.AppImage
- Operating System and version: Void Linux x86_64

**Comment by bolatovumar** [comment]
Looks like Dropbox has a guide about this issue here: https://www.dropbox.com/developers/reference/developer-guide#production-approval

**Comment by bolatovumar** [comment]
Tried it as well and getting the same error:

![Screen Shot 2020-03-02 at 7 55 25 PM](https://user-images.githubusercontent.com/1934678/75741687-39b71f00-5cc0-11ea-9f67-6ac3fa15cdb1.png)

![Screen Shot 2020-03-02 at 7 47 29 PM](https://user-images.githubusercontent.com/1934678/75741693-3e7bd300-5cc0-11ea-9a2f-438df802dced.png)

I managed to get the Dropbox backup working by creating my own app in Dropbox and replacing the Dropbox `clientId` in `default.js` with my newly created app's id. Looks like the existing Zap Dropbox and GDrive apps just need to be renewed or something. Ping @korhaliv @mrfelton @JimmyMow.
